aim/db/migration/alembic_migrations/versions/1b58ffa871bb_add_host_column.py

```diff
@@ -36,19 +36,19 @@
 def upgrade():
     op.add_column(
         'aim_endpoint_group_static_paths',
-        sa.Column('host', sa.String(512), nullable=True, index=True)
+        sa.Column('host', sa.String(256), nullable=True, index=True)
```

Comment:
should be 255 across all the changes.

Reply:
Also there is another migration script needs to be changed I think.

Reply:
Good catches -- will fix both.

Reply:
Just pushed changes to address the reviewer comments.